recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -35,11 +35,11 @@ PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'thunder_releas
 
 # Buildtype
 # Maybe we need to couple this to a Yocto feature
-PACKAGECONFIG[debug] = "-DBUILD_TYPE=Debug,,"
```

Comment:
I believe we need a backward compatibility here. Builds for older framework won't break this way